src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeProtocol.cs

```diff
@@ -17,26 +19,31 @@ public static class HandshakeProtocol
     {
         private const string ProtocolPropertyName = "protocol";
         private const string ProtocolVersionPropertyName = "version";
+        private const string MinorVersionPropertyName = "minorVersion";
         private const string ErrorPropertyName = "error";
         private const string TypePropertyName = "type";
 
-        /// <summary>
-        /// The serialized representation of a success handshake.
-        /// </summary>
-        public static ReadOnlyMemory<byte> SuccessHandshakeData;
```

Comment:
@muratg for breaking change approval in **3.0**. We're removing this since it's not really appropriate any more. It used to be the case that a successful handshake is always the exact same content. However, now it can vary based on the minor version provided. So, it's no longer appropriate to have a static field with the handshake content.

We could look at keeping this in as `Obsolete` but this is an extremely esoteric API that normal users shouldn't really be using (it should have been `internal` in the first place).

Reply:
I'm OK with removing in 3.0.

Question is, should we `Obsolete` it in 2.2?